src/Symfony/Bridge/Doctrine/DataCollector/DoctrineDataCollector.php

```diff
@@ -159,7 +159,9 @@ private function sanitizeQuery($connectionName, $query)
     private function sanitizeParam($var)
     {
         if (is_object($var)) {
-            return array(sprintf('Object(%s)', get_class($var)), false);
+            return method_exists($var, '__toString') ?
+                array($var->__toString(), false) :
```

Comment:
I agree that the class name should be dumped here as well. The way it is implemented will cause confusing results.